### PR TITLE
Module assignment status was incorrectly looking at External levels

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: dashboard/test/models/plc/enrollment_module_assignment_test.rb [hunk 1]
```diff
@@ -5,8 +5,13 @@ class Plc::EnrollmentModuleAssignmentTest < ActiveSupport::TestCase
     course = create :plc_course
     @course_unit = create(:plc_course_unit, plc_course: course)
     learning_module = create(:plc_learning_module, plc_course_unit: @course_unit)
-    @task1 = create(:plc_task, plc_learning_modules: [learning_module])
-    @task2 = create(:plc_task, plc_learning_modules: [learning_module])
+    @level1 = create(:external_link, url: 'some url')
```
Comment (aoby) [inline dashboard/test/models/plc/enrollment_module_assignment_test.rb:8]: add an external level as well to test that code path.

**Comment by aoby** [comment]
LGTM w minor test suggestion

File: dashboard/app/models/plc/enrollment_module_assignment.rb [hunk 2]
```diff
@@ -31,12 +31,14 @@ class Plc::EnrollmentModuleAssignment < ActiveRecord::Base
...
   def status
-    levels_tracked = UserLevel.where(user: user, level: plc_learning_module.stage.script_levels.map(&:levels).flatten)
-    passed_levels = levels_tracked.passing
+    levels_tracked = plc_learning_module.stage.script_levels.map(&:levels).flatten
+    levels_tracked.delete_if {|level| [External, ExternalLink].include? level.class}
+    user_progress_on_tracked_levels = UserLevel.where(user: user, level: levels_tracked)
+    passed_levels = user_progress_on_tracked_levels.passing
 
-    if levels_tracked.empty?
+    if user_progress_on_tracked_levels.empty?
```
Comment (aoby) [inline dashboard/app/models/plc/enrollment_module_assignment.rb]: Note if the levels are all external, the status will always be NOT_STARTED. Is this expected?

Comment (mehalshah) [inline dashboard/app/models/plc/enrollment_module_assignment.rb]: It's not. I'm going to switch the conditional order

**Pull request merged**